canonicalwebteam/flask_base/app.py

```diff
@@ -17,32 +21,71 @@ def __init__(
         self,
         name,
         service,
-        prepare_deleted=prepare_deleted(),
-        prepare_redirects=prepare_redirects(),
+        favicon_url=None,
+        template_404=None,
+        template_500=None,
         *args,
         **kwargs
     ):
         super().__init__(name, *args, **kwargs)
 
         self.service = service
 
-        self.config.from_object(
-            "canonicalwebteam.flask_base.config.BaseConfig"
-        )
+        self.config["SECRET_KEY"] = os.getenv("SECRET_KEY", "base_secret")
 
         self.url_map.strict_slashes = False
         self.url_map.converters["regex"] = RegexConverter
 
-        self.wsgi_app = (
-            ProxyFix(self.wsgi_app)
-            if not self.debug
-            else DebuggedApplication(self.wsgi_app)
-        )
+        self.wsgi_app = ProxyFix(self.wsgi_app)
 
-        self.before_request(prepare_redirects)
-        self.before_request(prepare_deleted)
+        if self.debug:
+            self.wsgi_app = DebuggedApplication(self.wsgi_app)
+
+        self.before_request(prepare_redirects())
+        self.before_request(
+            prepare_redirects(
+                path=os.path.join(self.root_path, "redirects.yaml"),
```

Comment:
Should path be "permanent_redirects.yaml" here?

Reply:
absolutely